Web/Edubase.Services/Nomenclature/NomenclatureService.cs

```diff
@@ -43,6 +43,7 @@ public static string GetEstablishmentsPluralName(GT groupType, eTextCase textCas
                 case GT.MultiacademyTrust:
                 case GT.SingleacademyTrust:
                 case GT.SchoolSponsor:
+                case GT.SecureSingleAcademyTrust:
```

Comment:
Here is a reference to SAT - does this need to be updated to also include Secure SAT?

https://github.com/DFE-Digital/get-information-about-schools/blob/secure_16-19_groups/Web/Edubase.Web.UI/Areas/Establishments/Controllers/EstablishmentController.cs#L655-L659

```csharp
        /// <summary>
        /// This all needs to be refactored, but not before a major release ;-)
        /// </summary>
        /// <param name="establishmentUrn"></param>
        /// <param name="parentGroups"></param>
        /// <param name="principal"></param>
        /// <returns></returns>
        internal static GroupModel GetLegalParent(int establishmentUrn, IEnumerable<GroupModel> parentGroups, IPrincipal principal)
        {
            var parentGroup = parentGroups.FirstOrDefault(g => g.GroupTypeId == (int) eLookupGroupType.SingleacademyTrust);
            if (parentGroup != null)
            {
                return parentGroup;
            }

            parentGroup = parentGroups.FirstOrDefault(g => g.GroupTypeId == (int) eLookupGroupType.MultiacademyTrust);
            if (parentGroup != null)
            {
                return parentGroup;
            }

            parentGroup = parentGroups.FirstOrDefault(g => g.GroupTypeId == (int) eLookupGroupType.Trust);
            return parentGroup ?? parentGroups.FirstOrDefault();
        }
```

Reply:
Thanks Roger.  Looks like this is something we didn't have any details on.  It could be a new story as it was not identified in any of the stories we have.

Reply:
Emailed Mark asking for a new story for this and to investigate the use of the enums in other parts of the code not directly linked to the current stories.